docs/api/apiv3/components/schemas/storage_file_model.yml

```diff
@@ -2,4 +2,36 @@
 ---
 allOf:
   - $ref: './file_link_origin_data_model.yml'
-  - $ref: './storage_file_location_model.yml'
+  - type: object
+    required:
+      - _type
+      - location
+      - _links
+    properties:
+      _type:
+        type: string
+        enum:
+          - StorageFile
+      location:
+        type: string
+        description: Location identification for file in storage
+      _links:
+        type: object
+        required:
+          - self
+        properties:
+          self:
+            allOf:
+              - $ref: './link.yml'
+              - description: |-
+                  Not provided
+
+                  **Resource**: urn:openproject-org:api:v3:storages:storage_file:no_link_provided
+example:
+  id: 42
+  name: readme.md
```

Comment:
can you put some more example data from the `file_link_origin_data_model`?

Reply:
Yes.